subgraph-api/reset.d.ts

```diff
@@ -0,0 +1,4 @@
+// unsure why but eslint can't recognize this is installed
+// and these 2 rules fail. Not ideal but..
+// eslint-disable-next-line node/no-missing-import
+import '@total-typescript/ts-reset/array-includes' // eslint-disable-line node/no-unpublished-import
```

Comment:
The linter considers dev dependencies as "unpublished". The idea is that even when you run "npm i --production", everything will work just fine.

In this case, it seems to be considering `ts-reset` is required at runtime, thus the error.